packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md

```diff
@@ -1,2 +1,2 @@
-  If you are using the sample app use: `http://localhost:8080`.
-  Otherwise, choose a URI that your app accepts.
+  If you are using the embedded authentication sample apps, add the following redirect URI `http://localhost:8080` (in addition to the default redirect URI `http://localhost:8080/authorization-code/callback` that is already listed).
```

Comment:
Please verify the redirect URI. Note that this default might change in future depending on how the backend Orgs are provisioned.

Reply:
```suggestion
  If you are using the embedded authentication sample apps, add `http://localhost:8080` as the redirect URI.
```

Reply:
looks good.

Reply:
Discussed with Arvin: decided to remove the text in brackets.